doc/dash/dash-sonic-hld.md

```diff
@@ -696,10 +711,17 @@ key                      = DASH_TUNNEL_TABLE:tunnel_name; tunnel name used for r
 ; field                  = value
 endpoints                = list of addresses for ecmp tunnel
 encap_type               = vxlan or nvgre
-vni                      = vni value for encap
+vni                      = vni value for encap, create only attribute
 metering_class_or        = uint32
 ```
 
+DASH_TUNNEL_TABLE shall have one or more endpoints. Encap type, VNI are create only attributes. A change on encap would require deleting and creating new tunnel objects. 
+One endpoint is treated as single nexthop and comma separated multiple endpoints shall be treated as ECMP nexthop.
+
+For single endpoint, implmentation shall simply create a sai_dash_tunnel object with ```SAI_DASH_TUNNEL_ATTR_DIP=endpoint IP``` and ```SAI_DASH_TUNNEL_ATTR_MAX_MEMBER_SIZE=1```
+
+For ECMP, implementation shall create ```sai_dash_tunnel_member``` and ```sai_dash_tunnel_next_hop``` with appropriate ```SAI_DASH_TUNNEL_ATTR_MAX_MEMBER_SIZE```
```

Comment:
Please mention that though members can be updated, at any point the number of members cannot exceed the count provided at Dash tunnel create

Reply:
addressed